_episodes/03-matplotlib.md

```diff
@@ -27,7 +27,7 @@ matplotlib.pyplot.show()
 ~~~
 {: .language-python}
 
-![Heatmap of the Data](../fig/inflammation-01-imshow.svg)
+![Heatmap representing the 'data' variable matrix. Each cell is coloured by value along a colour gradient from blue to yellow.](../fig/inflammation-01-imshow.svg)
```

Comment:
Couple of comments:
1. Our lesson uses American English -- please change colour to color.
2. I believe "heatmap" is jargon that we don't explain anywhere and we should. We can address it here or in a separate PR. What do you think?

Reply:
Thank you for the comments. I will make the spelling change.

I don't mind adding some text explaining a heatmap in this PR if you think it is appropriate, but I do think it should be outside the image description. Maybe in the text directly following this image?

Also, I have just noticed that the rest of the text uses 'heat map' rather than 'heatmap'. Should I modify the image description to make this consistent?

Reply:
> I don't mind adding some text explaining a heatmap in this PR if you think it is appropriate, but I do think it should be outside the image description. Maybe in the text directly following this image?

We can add an explanation to the Glossary (reference.md) and link to it in the main text.
I think it makes sense to do it in a separate PR but this is totally up to you.

> Also, I have just noticed that the rest of the text uses 'heat map' rather than 'heatmap'. Should I modify the image description to make this consistent?

That'd be great! I don't have a preference as to what spelling to use but we should use the same spelling throughout the lesson.

Reply:
> We can add an explanation to the Glossary (reference.md) and link to it in the main text.
I think it makes sense to do it in a separate PR but this is totally up to you.

I agree a separate PR would make more sense.